folder.md

```diff
@@ -49,113 +49,113 @@ The output is on the form:
 ]  
 ```
 
-The `permissions` key specify the access right your user has
+The `permissions` key specifies the access rights that your users have:
 
-| name        | Description         
+| Name        | Description         
 | ------------- |-------------
-| READ    | User can see the folder
-| ADMIN    | User can change the folder
-| WRITE    | Limit the list to folders with the parent ID specified
-| PERMANENT    | The folder is permanent (cannot be altered or deleted)
-| TRASH    | 	The folder is the user’s trashcan (implies PERMANENT)
-| PUBLIC | 	The folder is a public folder (implies PERMANENT)
+| READ    | User can see the folder.
+| ADMIN    | User can change the folder.
+| WRITE    | Limit the list to folders with the parent ID specified. [Is this correct?]
+| PERMANENT    | The folder is permanent (it cannot be altered or deleted).
+| TRASH    | 	The folder is the user’s trashcan (implies it is PERMANENT).
+| PUBLIC | 	The folder is a Public Media Gallery folder (implies it is PERMANENT).
 
 
 
-The endpoint supports the following filtering options (all optional):
+This endpoint supports the following filtering options (all optional):
 
 | Parameter        | Description         
 | ------------- |-------------
-| q    | Query string matching against the folder name 
-| id    | Limit the list to the folder that has the given ID. Multiple IDs can be given seperated by "+"
-| parent    | Limit the list to folders with the parent ID specified
-| sort_by    | Specifies which field you would like to sort by, currently `name` and `created` are supported.
-| sort_order    | Can be either `asc`, or `desc`. Default is `asc`
+| q    | Query the string matching the folder name. 
```

Comment:
We want to use the word query string (see https://en.wikipedia.org/wiki/Query_string)